### PR TITLE
Ddpb 1771

### PR DESCRIPTION
**Pull request by elvisciotti**

### CODE REVIEW

File: src/AppBundle/Service/CasrecVerificationService.php [hunk 1]
```diff
@@ -68,6 +71,19 @@ public function getLastMatchedDeputyNumbers()
         return $deputyNumbers;
     }
 
+    /**
+     * @return true if at least one matched CASREC contains NDR flag set to true
+     */
+    public function isLastMachedDeputNdrEnabled()
```
Comment (slizzio) [inline src/AppBundle/Service/CasrecVerificationService.php]: missing y in method name (deput Y)

File: src/AppBundle/Service/UserRegistrationService.php [hunk 2]
```diff
@@ -64,14 +66,15 @@ public function selfRegisterUser(SelfRegisterData $selfRegisterData)
...
+        $this->casrecVerificationService->validate( $selfRegisterData->getCaseNumber()
                                       , $selfRegisterData->getClientLastname()
                                       , $selfRegisterData->getLastname()
                                       , $user->getAddressPostcode()
                                       );
 
-        $user->setDeputyNo(implode(',', $this->casrecService->getLastMatchedDeputyNumbers()));
+        $user->setDeputyNo(implode(',', $this->casrecVerificationService->getLastMatchedDeputyNumbers()));
         $user->setCoDeputyClientConfirmed($isMultiDeputyCase);
+        $user->setOdrEnabled( $this->casrecVerificationService->isLastMachedDeputNdrEnabled());
```
Comment (slizzio) [inline src/AppBundle/Service/UserRegistrationService.php]: and here is the call  (missing y)

File: src/AppBundle/Service/CasrecVerificationService.php (hunk 1 above)
Comment (elvisciotti) [reply src/AppBundle/Service/CasrecVerificationService.php]: changed that

File: src/AppBundle/Service/UserRegistrationService.php (hunk 2 above)
Comment (elvisciotti) [reply src/AppBundle/Service/UserRegistrationService.php]: changed

**Comment by elvisciotti** [comment]
changes made

**Pull request merged**